Carta.py

```diff
@@ -320,33 +341,73 @@ def opponentRespondsInGrabPhase(self):
             else:
                 print("Carta.py: opponentRespondsInGrabPhase: opponent does not touch any card")
 
+    def removeCurrentGrabbingCard(self):
+        for i in range(len(self.grabbingCardsInPlay)):
+            if (self.grabbingCardsInPlay[i].getLastWord() == self.curReadingCard.getLastWord()):
+                self.grabbingCardsInPlay[i].setFrame(None)
+                self.grabbingCardsInPlay[i].setStatus(GrabCardStatus.INVALID)
+                self.usedGrabbingCards.cards.append(self.grabbingCardsInPlay.pop(i))
+
+    def shouldOpponentTransfer(self, yourFalseTouch):
```

Comment:
also make a `shouldYouTransfer`

even better, combine `shouldYouTransfer` and `shouldOpponentTransfer`